pkg/chartutil/save.go

```diff
@@ -99,7 +115,12 @@ func SaveDir(c *chart.Chart, dest string) error {
 // will generate /foo/bar-1.0.0.tgz.
 //
 // This returns the absolute path to the chart archive file.
-func Save(c *chart.Chart, outDir string) (string, error) {
+func Save(c *chart.Chart, outDir string, saveOpts ...SaveOpt) (string, error) {
```

Comment:
this breaks backwards compatibility.

My suggestion here would be to introduce a new function: `SaveWithOpts`.

Reply:
This has been addressed in the underlying PR (#8216 ). I will rebase this now to reflect those changes